### DISCUSSION TITLE
SCRIPT ERROR: @MISS-ELS/client/lights.lua:155: attempt to index a nil value (field '?' )

### DISCUSSION THREAD

**Discussion by LupoFG** [matsn0w/IIS-EmergencyLS-ELS-FiveM#144] [Q&A]
I have the following error and can't solve it with the information from line 155.
![image](https://github.com/matsn0w/MISS-ELS/assets/139262724/99a349af-32b6-4112-ab6f-29eac7b3e4b7)

**Comment by matsn0w** [comment]
Hi there, can you please provide your VCF for the relevant vehicle(s)?

**Comment by LupoFG** [comment]
<?xml version="1.0" encoding="utf-8" ?>

<vcfroot Description="EmergencyLS | VCF sample configuration" Author="Infinity Studios">
	
	<EOVERRIDE>	
		<Extra01 IsElsControlled="true" AllowEnvLight="true" Color="blue" OffsetX="-0.70" OffsetY="-0.20" OffsetZ="0.50" /> 
		<Extra02 IsElsControlled="true" AllowEnvLight="true" Color="blue" OffsetX="-0.25" OffsetY="-0.20" OffsetZ="0.50" />
		<Extra03 IsElsControlled="true" AllowEnvLight="true" Color="blue" OffsetX="0.25" OffsetY="-0.20" OffsetZ="0.50" />
		<Extra04 IsElsControlled="true" AllowEnvLight="true" Color="blue" OffsetX="0.70" OffsetY="-0.20" OffsetZ="0.50" />
		<Extra05 IsElsControlled="true" AllowEnvLight="true" Color="blue" OffsetX="-0.70" OffsetY="-0.20" OffsetZ="0.50" />
		<Extra06 IsElsControlled="true" AllowEnvLight="true" Color="blue" OffsetX="0.70" OffsetY="-0.20" OffsetZ="0.50" />
		<Extra07 IsElsControlled="true" AllowEnvLight="true" Color="red" OffsetX="-0.40" OffsetY="-0.80" OffsetZ="0.40" />
		<Extra08 IsElsControlled="true" AllowEnvLight="true" Color="red" OffsetX="0.00" OffsetY="-0.80" OffsetZ="0.40" />
		<Extra09 IsElsControlled="true" AllowEnvLight="true" Color="red" OffsetX="0.40" OffsetY="-0.80" OffsetZ="0.40" />
		<Extra10 IsElsControlled="false" />
		<Extra11 IsElsControlled="false" />
		<Extra12 IsElsControlled="false" AllowEnvLight="true" Color="white" />
	</EOVERRIDE>

	
	<STATIC>
		<Extra11 Name="Bullbar" />
		<Extra12 Name="Takedown lights" />
	</STATIC>

	
	<SOUNDS>
		<MainHorn InterruptsSiren="true" SoundBank="DLC_WMSIRENS/SIRENPACK_ONE" SoundSet="DLC_WMSIRENS_SOUNDSET" AudioString="SIRENS_AIRHORN" />
    		<ManTone1 AllowUse="true" SoundBank="DLC_WMSIRENS/SIRENPACK_ONE" SoundSet="DLC_WMSIRENS_SOUNDSET" AudioString="SIREN_ALPHA" />
    		<ManTone2 AllowUse="true" SoundBank="DLC_WMSIRENS/SIRENPACK_ONE" SoundSet="DLC_WMSIRENS_SOUNDSET" AudioString="SIREN_BRAVO" />
    		<SrnTone1 AllowUse="true" SoundBank="DLC_WMSIRENS/SIRENPACK_ONE" SoundSet="DLC_WMSIRENS_SOUNDSET" AudioString="SIREN_ALPHA" />
    		<SrnTone2 AllowUse="true" SoundBank="DLC_WMSIRENS/SIRENPACK_ONE" SoundSet="DLC_WMSIRENS_SOUNDSET" AudioString="SIREN_BRAVO" />
    		<SrnTone3 AllowUse="true" SoundBank="DLC_WMSIRENS/SIRENPACK_ONE" SoundSet="DLC_WMSIRENS_SOUNDSET" AudioString="SIREN_ALPHA" />
    		<SrnTone4 AllowUse="true" SoundBank="DLC_WMSIRENS/SIRENPACK_ONE" SoundSet="DLC_WMSIRENS_SOUNDSET" AudioString="SIREN_BRAVO" />
    		<AuxSiren AllowUse="false" AudioString="VEHICLES_HORNS_SIREN_1" />
    		<PanicMde AllowUse="false" AudioString="VEHICLES_HORNS_POLICE_WARNING" />	
	</SOUNDS>
	
	<PATTERN>
		<PRIMARY>
			<Flash01 Duration="80" Extras="1,2" /> 
			<Flash02 Duration="80" Extras="1,2,3,4" /> 
			<Flash03 Duration="80" Extras="3,4" /> 
			<Flash05 Duration="80" Extras="1,2" />
			<Flash04 Duration="80" />
			
		</PRIMARY>

		<SECONDARY> 
			<Flash01 Duration="280" Extras="6" />
			<Flash02 Duration="160" />
			<Flash03 Duration="280" Extras="5" />
			<Flash04 Duration="160" />
		</SECONDARY>

		<REARREDS IsEmergency="false"> 
			<Flash01 Duration="160" Extras="9" />
			<Flash02 Duration="160" />
			<Flash03 Duration="160" Extras="7" />
			<Flash04 Duration="160" />
		</REARREDS>
	</PATTERN>
</vcfroot>

**Reply by matsn0w** [reply]
Is this the entire file? You are missing some parts that are required. See the [docs](https://github.com/matsn0w/MISS-ELS/blob/main/docs/vcf.md#vehicle-configuration-file-vcf).
